core/src/main/java/pl/agh/edu/actor/component/table/CreditTable.java

```diff
@@ -0,0 +1,109 @@
+package pl.agh.edu.actor.component.table;
+
+import static pl.agh.edu.actor.component.table.CreditTable.CreditTableStyle.*;
+
+import java.util.Arrays;
+import java.util.stream.IntStream;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.Container;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.utils.Align;
+
+import pl.agh.edu.actor.GameSkin;
+import pl.agh.edu.actor.utils.FontType;
+import pl.agh.edu.actor.utils.LanguageLabel;
+import pl.agh.edu.actor.utils.LinkLabel;
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.management.bank.BankAccountHandler;
+import pl.agh.edu.model.bank.Credit;
+
+public class CreditTable extends BaseTable {
+	private final int noColumns = 3;
+	private final BankAccountHandler bankAccountHandler;
+
+	public CreditTable(BankAccountHandler bankAccountHandler) {
+		super();
+		this.bankAccountHandler = bankAccountHandler;
+		CreditBaseRow creditHeaderRow = new CreditBaseRow("creditTable.column.date", "creditTable.column.monthly", "creditTable.column.payall");
+		creditHeaderRow.align(Align.bottomLeft);
+		innerTable.add(creditHeaderRow).space(getRowSpacing()).growX().align(Align.bottomLeft);
+		innerTable.row();
+		for (Credit credit : bankAccountHandler.getCurrentCredits().keySet()) {
+			BaseRow row = new CreditBaseRow(credit);
+			row.align(Align.bottomLeft);
+			innerTable.add(row).space(getRowSpacing()).growX().align(Align.bottomLeft);
```

Comment:
może warto dodać metodkę do BaseTable add BaseRow aby nie mysieć mieć tu tego jeszcze można to skrócić do
```java
innerTable.add(row).space(getRowSpacing()).growX().align(Align.bottomLeft.row());
```